src/recipes/link.ts

```diff
@@ -0,0 +1,33 @@
+import { defineRecipe } from '@pandacss/dev'
+
+const linkRecipe = defineRecipe({
+  className: 'link',
+  description: '<a> styles',
+  base: {},
+  variants: {
+    in: {
+      modal: {
+        color: 'lightblue',
+      },
+      breadcrumbs: {
+        color: 'inherit',
+        textDecoration: 'none',
+      },
+      thoughtAnnotation: {
+        height: '1em',
+        display: 'inline-block',
+        overflow: 'hidden',
+        position: 'relative',
+        zIndex: 'thoughtAnnotationLink',
+      },
```

Comment:
The `breadcrumbs` and `thoughtAnnotation` variants were added to the recipe but unless I am missing something they are not used anywhere. Is that true? We should aim to replace the old App.css usage in the same commit as adding the new PandaCSS usage.

Reply:
I can look at that. I was planning on just moving the classes that affect a tags into the link recipe first, and deal with them later.

Reply:
the `Link` component is used within breadcrumbs I believe. I think looking in this is a bit more complex, so I'll leave this for now.